persistence.js

```diff
@@ -57,7 +58,7 @@ async function * retainedMessagesByPattern (db, pattern) {
 async function subscriptionsByClient (db, client) {
   const resubs = []
   for await (const sub of decodedDbValues(db, subByClientKey(client.id))) {
-    const resub = rmClientId(sub)
+    const { clientId, ...resub } = sub
```

Comment:
While I like this it's not really straightforward to read

```suggestion
    // remove clientId from sub
    const { clientId, ...resub } = sub
```